features/cli/policy.feature

```diff
@@ -463,63 +463,29 @@ Feature: change the policy of user/service account
     Then the step should succeed
     And the output should not match:
       | .*default.*restricted |
-    Given I obtain test data file "authorization/scc/PodSecurityPolicyReview.json"
-    Given I run the :policy_scc_review client command with:
-      | f | PodSecurityPolicyReview.json |
-      | n | <%= project.name %>                                                                            |
```

Comment:
@y4sht , @redHatGong, this PR also removes **UNNEEDED** checkpoints whose only difference is whether `| n              | <%= project.name %>` is added; we know `| n              | <%= project.name %>` is of course used after `oc new-project`, with and without -n <%= project.name %> don't have difference, so duplicate checkpoints are UNNEEDED.